### ISSUE TITLE
OpenShift Version is Out of Date

### ISSUE THREAD

**Issue by liam-middlebrook** [ryansb/ofCourse#10]
The version of ofCourse that is stored on OpenShift is out of date (by more than a week).We won't be able to rely on OpenShift to frequently update their Pip mirror every time we push out an update; so we should come up with an alternative.

We need some way to make sure that when we are deploying to OpenShift that the latest version of ofCourse is being ran. Is it possible that we could have OpenShift reference the canonical PyPi package distribution rather than their own?

**Comment by liam-middlebrook** [comment]
I currently have a branch on [my fork of ofCourse](https://github.com/liam-middlebrook/ofCourse/tree/openshift-git) that has attempts to use the `dependency_links` in the auto-generated setup.py on deploy. Currently that doesn't seem to be working, but I think it's a step in the right direction.

**Comment by ryansb** [comment]
I'm pretty sure you have the wrong version format. You're using `git+ssh://git@github.com/ryansb/ofCourse.git#egg=ofcourse-0.0.0` but I think you need `git+git://github.com/ryansb/ofCourse.git@develop#egg=ofcourse` because anonymous SSH doesn't work (AFAIK) on github.

Also for your purposes you probably want to use `develop` instead of `master`

**Comment by liam-middlebrook** [comment]
I figured out part of the issue with why my dependency_links weren't working. When `setup.py` was being called on deployment it already recognized that ofCourse was installed on that machine and it had no need to download a newer version. I deleted the apps and recreated them and it seems to work.

A potential fix (although I think there is a better way) would be to increment the version number that is required in the auto-generated files by the ofCourse openshift cli with each release.
